Data/Graph/Dom.hs

```diff
@@ -600,29 +605,3 @@ fetch :: (MArray (A z) a (ST z))
 fetch f i = do
   a <- gets f
   st (a!:i)
-
------------------------------------------------------------------------------
-
--- g0 = fromAdj
---   [(1,[2,3])
---   ,(2,[3])
---   ,(3,[4])
---   ,(4,[3,5,6])
---   ,(5,[7])
---   ,(6,[7])
---   ,(7,[4,8])
---   ,(8,[3,9,10])
---   ,(9,[1])
---   ,(10,[7])]
-
--- g1 = fromAdj
---   [(0,[1])
---   ,(1,[2,3])
---   ,(2,[7])
---   ,(3,[4])
---   ,(4,[5,6])
---   ,(5,[7])
---   ,(6,[4])
---   ,(7,[])]
-
------------------------------------------------------------------------------
```

Comment:
Was there a reason to remove these comments? Depending on @AndreasPK's workflow, they might be used as debugging fixture...

Reply:
True. I just removed them because these are also verbatim, non-commented, in tests/Main.hs

Reply:
While in general sebastian is right in this specific case it's fine.